client/client.go

```diff
@@ -388,19 +390,87 @@ func createClientWithKeyspace(
 	return c, nil
 }
 
-// NewClientWithKeyspaceName creates a client with context and the specified keyspace name.
-func NewClientWithKeyspaceName(
+// APIVersion is the API version the server and the client is using.
+// See more details in https://github.com/tikv/rfcs/blob/master/text/0069-api-v2.md#kvproto
+type APIVersion int
+
+// The API versions the client supports.
+const (
+	V1 APIVersion = iota
+	V1TTL
```

Comment:
What's the difference between V1 and V1TTL? So far, this PR doesn't reflect any differentiation. Do we intend to keep V1TTL here in case V1TTL might be different from V1 in NewClientWithAPIContext in the future?

Reply:
I think so. /cc @iosmanthus

Reply:
This reminds me that V1TTL is not designed for clients, we should remove this from the implementation. However, the value of the APIVersion should be synced with kvproto.